Resources/public/js/views/universaldiscovery/ez-universaldiscoveryfinderexplorerview.js

```diff
@@ -49,6 +49,11 @@ YUI.add('ez-universaldiscoveryfinderexplorerview', function (Y) {
                         this._addLevel(location);
                     }
                 }, this);
+                if (this.get('discoverRootDepth')) {
+                    for (var i = 0; i < this.get('discoverRootDepth'); i++) {
```

Comment:
this is weird, this should be integrated in the loop above. That also means eZ.UniversalDiscoveryFinderExplorerLevelView should subscribed to `disabledChange` but it should expect the value to be set at build time.